QML_HUD/gauge.cpp

```
@@ -0,0 +1,38 @@
+
+Gauge::Gauge(QQuickItem* parent)
+    : AbstractControl(parent)
+{
+}
+
+void Gauge::paint(QPainter* painter)
+{
+    qreal m_MinValue = 0;
+    qreal m_MaxValue = 360;
+    qreal spanAngle = -360;
+    qreal offset = 5;
```

cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_PROPERTY is a macro then please configure it.

(unknownMacro)